src/Providers/Maps/Google/SearchPlaces/SearchPlaces.ts

```diff
@@ -63,7 +65,48 @@ namespace Provider.Maps.Google.SearchPlaces {
 				);
 			}
 
-			if (typeof google === 'object' && typeof google.maps === 'object') {
+			if (typeof google.maps.places === 'object') {
```

Comment:
Why did we stop testing for `typeof google === 'object' && typeof google.maps === 'object'` if any of them is undefined wouldn't this be a problem? Or you think that it's not an issue?

Reply:
This is tested on initializeScripts method that will call this method;